net/src/packet/mod.rs

```diff
@@ -133,10 +138,7 @@ impl<Buf: PacketBufferMut> Packet<Buf> {
     /// ```
     pub fn vxlan_decap(&mut self) -> Option<Result<Vxlan, ParseError<EthError>>> {
         match self.headers.try_vxlan() {
-            None => {
```

Comment:
why are we removing the log here?

Reply:
We call vxlan_decap for every packet destined to the gateway, but the gateway may receive packets that are not vxlan (control-plane, icmp, etc..). So the prior log would show for every packet sent to it.